### PR TITLE
Documentation of Source Code

### PR DESCRIPTION
**Pull request by aviiciii**
### Code Documentation
To make the code easier for other developers to comprehend, I have provided brief descriptions of the code snippets in the HTML, CSS, and Javascript files.

This pull request solves #91 .

**Code Snippets:**
### HTML:
d2
d3
### Javascript:
d1

### CODE REVIEW

File: assets/js/dark.js [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
-    }else{
-      document.getElementById(iconID).className = "bx bx-moon dark-btn";
-    }
-  }
+	if(document.getElementById(iconID).className=="bx bx-moon dark-btn"){
+		document.getElementById(iconID).className = "bx bx-sun dark-btn";
+	}else{
+		document.getElementById(iconID).className = "bx bx-moon dark-btn";
+	}
+}
```
Comment (arpitghura) [inline assets/js/dark.js:9]: Fix the spacing

**Review by arpitghura — CHANGES_REQUESTED** [review]
You also not added the comments in `testimonials.css`

File: assets/js/dark.js (hunk 1 above)
Comment (aviiciii) [reply assets/js/dark.js]: Done both the tasks successfully.

**Comment by aviiciii** [comment]
@arpitghura I have completed the requested reviews and commited to this pull request. Kindly review the changes and accept the pull request if no further changes are required.

**Review by arpitghura — APPROVED** [review]
looks good

**Comment by arpitghura** [comment]
Do resolve the merge conflicts. If you face any issue, I am happy to help.

**Comment by aviiciii** [comment]
@arpitghura I have resolved all merge conflicts. Kindly proceed with the pull request merge.

**Review by arpitghura — CHANGES_REQUESTED** [review]
The icons in Navigation menu are not visible on practice page. Have a look at this issue.
image

Do the required changes and let me know if you need any help.

**Comment by aviiciii** [comment]
@arpitghura I have fixed the navigation bar icons. Kindly accept the pull request.

**Pull request merged**